src/test/java/com/google/sps/data/friend_map/FriendsMapCreationTest.java

```diff
@@ -74,48 +71,52 @@ public void oneUserGraphTest() {
   */
   @Test
   public void twoUsersNoConnectionsGraphTest() {
-    userA = new UserNode(USER_A_ID, ImmutableSet.of());
-    userB = new UserNode(USER_B_ID, ImmutableSet.of());
+    UserNode userA = new UserNode(USER_A_ID, ImmutableSet.of());
+    UserNode userB = new UserNode(USER_B_ID, ImmutableSet.of());
 
     Set<UserNode> twoUserSet = new HashSet<>();
     twoUserSet.add(userA);
     twoUserSet.add(userB);
     UserFriendsMap twoUserFriendsMap = new UserFriendsMap(twoUserSet);
 
-    Map<String, ImmutableSet<String>> expectedMap = new HashMap<>();
-    expectedMap.put(USER_A_ID, ImmutableSet.of());
-    expectedMap.put(USER_B_ID, ImmutableSet.of());
+    ImmutableMap<String, ImmutableSet<String>> expectedMap =
+      ImmutableMap.of(
+        USER_A_ID, ImmutableSet.of(),
+        USER_B_ID, ImmutableSet.of()
+      );
 
-    Assert.assertEquals(twoUserFriendsMap.getFriendMap(), ImmutableMap.copyOf(expectedMap));
+    assertThat(twoUserFriendsMap.getFriendMap(), is(expectedMap));
   }
 
   /**
   * Tests if map is forming correctly when there are just 3 users, 2 of which are
   * friends with the third user. There are no connections other than this.
   *
   * <p>Should result in a UserFriendsMap with three keys:
-  *    User C should map to a friend set with just User E
-  *    User D should map to a friend set with just User E
-  *    User E should map to a friend set with User C and User D.
+  *    User A should map to a friend set with just User C
+  *    User B should map to a friend set with just User C
+  *    User C should map to a friend set with User A and User B.
   */
   @Test
   public void threeUsersTwoConnectionGraphTest() {
-    userC = new UserNode(USER_C_ID, ImmutableSet.of(USER_E_ID));
-    userD = new UserNode(USER_D_ID, ImmutableSet.of(USER_E_ID));
-    userE = new UserNode(USER_E_ID, ImmutableSet.of(USER_C_ID, USER_D_ID));
+    UserNode userA = new UserNode(USER_A_ID, ImmutableSet.of(USER_C_ID));
+    UserNode userB = new UserNode(USER_B_ID, ImmutableSet.of(USER_C_ID));
+    UserNode userC = new UserNode(USER_C_ID, ImmutableSet.of(USER_A_ID, USER_B_ID));
     
     Set<UserNode> threeUsersTwoConnectionSet = new HashSet<>();
```

Comment:
See earlier comment about set construction.

Reply:
Got it!